tests/by-util/test_cksum.rs

```diff
@@ -470,6 +470,17 @@ fn test_binary_file() {
         .arg(at.subdir.join("f"))
         .succeeds()
         .stdout_contains("d41d8cd98f00b204e9800998ecf8427e *");
+
+    // no "*" in this case
+    scene
+        .ucmd()
+        .arg("--untagged")
+        .arg("--tag")
```

Comment:
~~The reason why there is no "*" in this case is because `--tag` is last, and therefore the output format is tagged, which does not annotate binary-ness:~~

```console
$ ../gnu/src/cksum --tag -b --algo=md5 target/debug/coreutils
MD5 (target/debug/coreutils) = f70b38fd8e152eda7da97dd6dd0c7d76
$ ../gnu/src/cksum --tag --algo=md5 target/debug/coreutils
MD5 (target/debug/coreutils) = f70b38fd8e152eda7da97dd6dd0c7d76
```

EDIT: Ah, this is fixed by a later commit.